content/030_eksctl/launcheks.md

```diff
@@ -9,13 +9,18 @@ weight: 20
 **DO NOT PROCEED** with this step unless you have [validated the IAM role](/020_prerequisites/workspaceiam/#validate-the-iam-role) in use by the Cloud9 IDE. You will not be able to run the necessary kubectl commands in the later modules unless the EKS cluster is built using the IAM role.
 {{% /notice %}}
 
-#### Challenge:
+#### Challenge
+
 **How do I check the IAM role on the workspace?**
 
 {{%expand "Expand here to see the solution" %}}
-Run `aws sts get-caller-identity` and validate that your _Arn_ contains `eksworkshop-admin`and an Instance Id.
+Run the command below and validate that your _Arn_ contains `eksworkshop-admin`and an Instance Id.
+
+```bash
+aws sts get-caller-identity
+```
 
```

Comment:
This should actually probably be changed to the "output" shortcode

Reply:
Tx for the feedback, I just pushed the fix into this branch